data/json/items/generic.json

```diff
@@ -2658,6 +2676,20 @@
     ],
     "ammo": "money"
   },
+  {
+    "id": "robofac_test_data",
+    "type": "TOOL",
+    "symbol": ",",
+    "color": "yellow",
+    "name": "prototype I/O recorder",
+    "description": "This small transparent card was attached to the prototype robot's CPU.  It might contain the data the intercom spoke off.",
```

Comment:
```suggestion
    "description": "This small transparent card was attached to the prototype robot's CPU.  It might contain the data the intercom spoke of.",
```